src/css/custom.scss

```diff
@@ -449,11 +449,34 @@ $color-primary: #ff00a0;
     --prism-background-color: red;
   }
 
-  .table {
-    background-color: $dark-color;
-    border-radius: var(--ifm-pre-border-radius);
+  /* TABLE DARK MODE */
+
+  table {
+    border-collapse: collapse;
+    width: 100%;
+    table-layout: fixed;
```

Comment:
Can we change the table layout to auto (Since this is already browser default we can remove it in general). Fixed adds this unnecessary space. 
```suggestion

```
<img width="883" alt="Screenshot 2024-03-05 at 04 28 31" src="https://github.com/surrealdb/docs.surrealdb.com/assets/35943047/1ee4536c-87be-4e13-8d14-5d9f68edf4d9">

As opposed to: 
<img width="538" alt="Screenshot 2024-03-05 at 06 05 43" src="https://github.com/surrealdb/docs.surrealdb.com/assets/35943047/abd8b7c2-99e0-430a-8a9b-0576e33f0e19">